tasks.py

```diff
@@ -160,7 +160,7 @@ def restart(context):
 @task
 def stop(context):
     """Stop Nautobot and its dependencies."""
-    print("Stopping Nautobot...")
+    print("Stopping Nautobot (deleting containers)...")
```

Comment:
My only concern with this phrasing is that someone new to Docker might think they're losing data when this is done due to the "delete" word being used. I'd say we should probably have some documentation too around why someone might want to use down vs stop, etc.

Reply:
I suppose you are right, would you like to include something in the comments at line 162? I did get confused when my containers got deleted when using this option (I probably should have read the doc more closely as well). 
Or we would modify the message adding info about the volumes, for example:
`     print("Stopping Nautobot (deleting containers but keeping volumes and data)...")`

Reply:
I think it'd make sense to alter the console message to be clearer about what's happening and also documenting what the various invoke task options are. i know we're currently missing the later at this time.

Reply:
I agree. I am not sure how to help move this forward though. About presenting the options I think a help message should be defined in a central point in order to avoid repeating it for each option, possibly through a function (preferably) or a global variable. How can I help?